src/app/components/sidenav/sidenav.component.html

```diff
@@ -0,0 +1,54 @@
+<mat-sidenav-container class="sidenav-container" autosize>
+    <mat-sidenav [class.item-description]="isDescription" class="sidenav" mode="side" opened>
+
+        <div class="sidenav-content" >
+            <div class="primary-menu-items">
+                <button mat-icon-button (click)="toggleSidebar()">
+                    <mat-icon class="icon-style">menu</mat-icon>
+                </button>
+
+                <ng-container *ngIf="User==eRole.User">
```

Comment:
Try to avoid abbreviating variable names. It can create confusions. It's better use meaningful names 
example: `eRole  could be `roleType`
It looks like User actually holds a type; update the variable name accordingly.